src/BloomExe/web/ServerBase.cs

```diff
@@ -106,6 +106,7 @@ private void TryStart()
 			for (var i = 0; i < _workers.Length; i++)
 			{
 				_workers[i] = new Thread(RequestProcessorLoop);
+				_workers[i].Priority = ThreadPriority.BelowNormal;
```

Comment:
Updated so priority is BelowNormal only when processing a request for a thumbnail.